### ISSUE TITLE
Accessibility statement:

### ISSUE THREAD

**Issue by auxfuse** [StephenJ2020/soda-apply#31]
* static template
* default render function
* linked from the footer as a quick link

**Comment by StephenJ2020** [comment]
Template used for Initial Accessibility Statement, but could and should be improved with suggestions below.
https://www.nomensa.com/blog/writing-an-accessibility-statement

Great guide to what information should actually be contained in an Accessibility Statement:
https://www.hassellinclusion.com/blog/write-accessibility-statement/
- The role of an accessibility policy statement was to let the disabled users of a website know the important aspects of the way the site had been created to be accessible, and give details on how they could optimise their website user experience.
- Statements often read like a combination of a sales piece on how socially responsible the organisation is, a technology manual, and some legal small print.
- But these statements are supposed ‘to give accessibility information to disabled people’

Let site Users know how to report accessibility issue so that they can be resolved quicker!
https://www.w3.org/WAI/teach-advocate/contact-inaccessible-websites/
